gunicorn/config.py

```diff
@@ -883,12 +883,14 @@ class ReloadEngine(Setting):
 
 class ReloadExtraFiles(Setting):
     name = "reload_extra_files"
-    action = "append"
+    action = "store"
     section = "Debugging"
     cli = ["--reload-extra-file"]
     meta = "FILES"
     validator = validate_list_of_existing_files
-    default = []
+    default = ""
```

Comment:
@firekim2 bump on this question.

Reply:
bump firekim2

Reply:
So really sorry for not following up about this for this long time. 
Yes, it is. Since, we are changing the type to str, and parse it. it should be the empty string.

Reply:
I just reviewed the ArgumentParser documentation and tried this a bit in my REPL. Using `nargs='+'` causes the value to a be a list, so I think we want `[]` as the default.